core/templates/dev/head/components/OutcomeDestinationEditorDirective.js

```diff
@@ -37,7 +37,7 @@ oppia.directive('outcomeDestinationEditor', [
             EXPLORATION_AND_SKILL_ID_PATTERN, PLACEHOLDER_OUTCOME_DEST) {
           var currentStateName = null;
           $scope.canAddPrerequisiteSkill =
-            constants.ENABLE_NEW_STRUCTURE_EDITORS;
+            StateEditorService.getCanAddPrerequisiteSkill();
```

Comment:
Should it be: "constants.ENABLE_NEW_STRUCTURE_EDITORS && isExplorationWhitelisted()"? I.e. if we are not removing the "enable" flag yet we should keep it everywhere, and remove it "all at once" later.

Ditto elsewhere.

Btw, just to check, refresher exps are also behind a whitelist flag, right?

Reply:
Ok.
For refresher explorations, I think the restriction is that only admins/moderators can add them. Other than that, they can be added to any exploration.

Reply:
Hm, I think that's OK (though please check that the restriction you cited is actually the case). We plan to remove that feature in the long run anyway so I guess it's up to you whether to migrate it or not to the new flag.

Reply:
Yes, that is the case, as seen [here](https://github.com/oppia/oppia/blob/4a1046a3eca28b042679bca39333c9a38ee7455e/core/templates/dev/head/components/OutcomeDestinationEditorDirective.js#L61), where permission to add refresher exploration is restricted to only admins and moderators.

Reply:
Done initial comment. Added new structures flag check as well.